### PR TITLE
menu keybindings in default file

### PR DESCRIPTION
**Pull request by elferherrera**
# Description
Menu keybindings are added in the default config file. 

Note: Without defining the menu events in your config file, the menus will not appear 

# Tests

Make sure you've run and fixed any issues with these commands:

- [x] `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
- [x] `cargo clippy --all --all-features -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
- [x] `cargo build; cargo test --all --all-features` to check that all the tests pass

### CODE REVIEW

**Comment by fdncred** [comment]
@elferherrera I changed these to work on Mac. This is how I have them set. I think the first one may be the same, now that I think about it.
```
    {
       name: trigger-history-menu
       modifier: control
       keycode: char_x
       mode: emacs
       event: { 
         until: [
           { send: menu name: history_menu }
           { send: menupagenext }
         ]
       }
    }
    # ctrl+shift+x isn't even seen on mac
    {
       name: trigger-history-previous
       modifier: "control"
       keycode: char_z
       mode: emacs
       event: { 
         until: [
           { send: menu name: history_menu }
           { send: menupageprevious }
         ]
       }
    }
```

**Comment by elferherrera** [comment]
@fdncred we had ctr+X because JT set it like that. I actually changed it to something else in my config. What do people suggest as default for the file?

**Comment by fdncred** [comment]
Ctrl X is fine. Shift Ctrl X doesn't work on Mac.

**Comment by elferherrera** [comment]
With the changes to Reedline now the events are correctly identified. We can merge this one

File: src/reedline_config.rs [hunk 1]
```diff
@@ -168,17 +136,6 @@ pub(crate) fn create_keybindings(config: &Config) -> Result<KeybindingsMode, She
...
         "emacs" => {
             let mut keybindings = default_emacs_keybindings();
-            add_menu_keybindings(&mut keybindings);
-
-            // temporal keybinding with multiple events
-            keybindings.add_binding(
-                KeyModifiers::SHIFT,
-                KeyCode::BackTab,
-                ReedlineEvent::Multiple(vec![
-                    ReedlineEvent::Edit(vec![EditCommand::InsertChar('p')]),
```
Comment (jntrnr) [inline src/reedline_config.rs:178]: Am I losing my quick shell change? That's probably okay, we'll have enough examples folks can make one

Comment (elferherrera) [reply src/reedline_config.rs:178]: Oohhh, was that yours? Do you want me to add it back? You can create a keybinding for it

**Comment by fdncred** [comment]
@elferherrera I noticed you changed "previous" to control+z, thanks. I do have a question though, why is it not necessary to have an `until` part in "previous" but it's required in "next"?

**Comment by elferherrera** [comment]
@fdncred because previous only sends one event. If you would like that that keybinding be related to multiple actions then you would use an until

File: src/reedline_config.rs (hunk 1 above)
Comment (jntrnr) [reply src/reedline_config.rs:178]: I'll just add it to my personal one. It's not hard for other folks to add stuff like that

**Comment by jntrnr** [comment]
looks good

**Pull request merged**